cmake/FindLIBFABRIC.cmake

```diff
@@ -0,0 +1,15 @@
+# Copyright 2016 Thorsten Schuett <schuett@zib.de>, Farouk Salem <salem@zib.de>
+
+find_path(LIBFABRIC_INCLUDE_DIR rdma/fabric.h)
+find_library(LIBFABRIC_LIBRARY NAMES fabric)
+
+set(LIBFABRIC_LIBRARIES ${LIBFABRIC_LIBRARY} )
+set(LIBFABRIC_INCLUDE_DIRS ${LIBFABRIC_INCLUDE_DIR} )
+
+MESSAGE(STATUS "libfabric           = ${LIBFABRIC_LIBRARIES}")
+MESSAGE(STATUS "libfabric           = ${LIBFABRIC_INCLUDE_DIRS}")
+
+include_directories(SYSTEM ${LIBFABRIC_INCLUDE_DIRS})
```

Comment:
Suggest switching to per-target definitions (as we recently did in the rest of the build system) for better modularity: target_include_directories()...